nsepython/rahu.py

```diff
@@ -1,4 +1,6 @@
 import os,sys
+import warnings
+warnings.simplefilter(action='ignore',category=FutureWarning)
```

Comment:
@Priyansh-666 why do you want this? any specific examples that you are seeing with this?

Reply:
There is an unnecessary error returned by Pandas , and it clogs up the terminal , same with the printing by the nsepython fetch commands

I have added verbose and suppressed warnings because the terminal's memory gets full and make the code to crash